.github/workflows/book.yml

```diff
@@ -15,16 +15,27 @@ jobs:
           toolchain: nightly
           override: true
 
-      - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: 'latest'
+      # - name: Setup mdBook
+      #   uses: peaceiris/actions-mdbook@v1
+      #   with:
+      #     mdbook-version: 'latest'
+
+      # mdBook for mdbook-pdf compatibility by fixing broken links in print.html
+      # Waiting for https://github.com/rust-lang/mdBook/pull/1738 to get merged
+      - name: Clone patched mdbook source code
+        run: cd .. && git clone https://github.com/HollowMan6/mdBook
```

Comment:
This is unsafe for us, as it enables you to subsequently alter your `master` branch to run whatever you want on our CI 🙂

I'd be fine with this if instead it pins a specific revision corresponding to the mdBook PR.

Reply:
OK. I didn't mean to do something bad here... But what you mentioned is reasonable.